lib/network/modules/components/nodes/util/NodeBase.js

```diff
@@ -89,6 +90,23 @@ class NodeBase {
       }
     }
   }
+
+
+  /**
+   * Determine if the shape of a node needs to be recalculated.
+   *
+   * @protected
+   */
+  needRefresh(selected, hover) {
```

Comment:
> There are no guidelines

Actually, I think there should be. In due time, I will make a proposal; there are a couple of style issues that bother me.

> needsRefresh

Yes, that's better. Silly me.